### DISCUSSION TITLE
PaoPaoGW Fast

### DISCUSSION THREAD

**Discussion by wwushi** [kkkgo/PaoPaoGateWay#130] [问和答] [answered]
网关拉取订阅报错PaoPaoGW Fast 手机APP能正常拉取
![Snipaste_2024-09-21_15-24-12](https://github.com/user-attachments/assets/54fb17ab-06a5-4871-bd44-7726054a89e2)

**Comment by kkkgo** [comment]
Fast 是你的设置fast_node=yes在测试节点延迟

**Reply by wwushi** [reply]
默认3000没改过，上午还正常，subtime设置的一天，还没到重新拉取的时间，我试试更改一下fast_node的时间

**Reply by kkkgo** [reply, marked as answer]
cpudelay是并发测速间隔，和超时时间无关。
超时会从低到高标准测试，低延迟标准测试不通过timeout正常，并非拉取订阅失败。
超时时间会逐渐提升到最大5秒，如果5秒都全部节点超时，则会重置网关强制重新拉取所有配置。

**Reply by wwushi** [reply]
没事了，找到问题了，电信的锅，打扰了。。。。。。。尴尬。。。。。
